compiler/rustc_trait_selection/src/traits/error_reporting/mod.rs

```diff
@@ -947,7 +942,7 @@ impl<'tcx> TypeErrCtxtExt<'tcx> for TypeErrCtxt<'_, 'tcx> {
                             if !self.report_similar_impl_candidates(
```

Comment:
Follow-up: can `report_similar_impl_candidates` take a `LocalDefId`?

Reply:
yep, added a fixme to include it in a folloup PR